ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+### Context
+
+* [ ] **mapbox-gl-js version**: 
+* [ ] **browser version**: 
+* [ ] **operating system**: 
+
+### Report
+
+**Expected behavior**:
+
+**Actual behavior**:
+
+**Steps to reproduce the behavior**:
+
+1.
+2.
+3.
```

Comment:
About 90% of the bugs I've seen in this repo are most clearly described by a functioning minimal demo (i.e. in jsbin). I propose we keep this section more open-ended, with helpful guidelines: